src/service/resources-impl.js

```diff
@@ -1113,32 +1115,41 @@ export class Resources {
   /**
    * Schedules change of the element's height.
    * @param {!Resource} resource
-   * @param {number} newHeight
+   * @param {number|undefined} newHeight
+   * @param {number|undefined} newWidth
    * @param {boolean} force
    * @param {function=} opt_callback A callback function.
    * @private
    */
-  scheduleChangeHeight_(resource, newHeight, force, opt_callback) {
-    resource.resetPendingChangeHeight();
-    if (resource.getLayoutBox().height == newHeight) {
+  scheduleChangeSize_(resource, newHeight, newWidth, force, opt_callback) {
+    resource.resetPendingChangeSize();
```

Comment:
Let's either noop or assert that either newHeight or newWidth should be defined.